ui/raidboss/data/06-ew/raid/p8s.ts

```diff
@@ -10,14 +10,42 @@ import { PluginCombatantState } from '../../../../../types/event';
 import { NetMatches } from '../../../../../types/net_matches';
 import { TriggerSet } from '../../../../../types/trigger';
 
+// TODO: call out gorgon spawn locations
+// TODO: call out shriek specifically again when debuff soon? (or maybe even gaze/poison/stack too?)
+// TODO: crush/impact directions during 2nd beast phase
+// TODO: make the torch call say left/right during 2nd beast
+// TODO: better vent callouts
+// TODO: initial tank auto call on final boss as soon as boss pulled
```

Comment:
This is done no?

Reply:
No, there's a workaround to prevent a belated call after a few autos have come out.  There really needs to be some "the fight has started, get ready for auto in 5s" sorta call.